backend/resolvers.ts

```diff
@@ -1,57 +1,34 @@
 import { AppDataSource } from '@crossword/db';
 import { Crossword as CrosswordEntity} from './entities/crossword';
-import { Resolvers, Test, Crossword } from './generated/graphql';
+import { Resolvers, Test, Crossword, Point } from './generated/graphql';
 
 export const resolvers: Resolvers = {
   Query: {
-    test: (): Test => {
-      return {
-        name: 'ethan is resolved',
-        id: 'balls',
-      };
-    },
     crossword: async (): Promise<Crossword> => {
-      let name : string = (await AppDataSource.manager.findOneBy(CrosswordEntity,{name: "cringe"})).name
+      let crossword : CrosswordEntity = (await AppDataSource.manager.findOneBy(CrosswordEntity,{name: "cringe"}))
+      console.log(crossword)
```

Comment:
sussy

Reply:
to be clear this is referencing the console.log statement, not sure if it's intended or not

Reply:
done